deployments/spanish_language_next_word.yaml

```diff
@@ -0,0 +1,74 @@
+status: Draft
+registry_authors:
+  - Elena Ghazi
+  - Nicolas Berrios
+  - Jack Fitzsimons
+tier: 3
+deployment:
+  name: Spanish Next Word Prediction
+  data_curator: Google
+  description: A machine learning model trained with federated learning for next‑word prediction on Spanish‑language Gboard.
+  intended_use: Power next-word-prediction for Spanish-language Gboard users.
+  data_product_type: Machine learning model
+  publication_date: '2022-02-01'
+  data_product_region: The model was specifically trained to power next-word prediction for Spanish-language Gboard users. The specific countries the users are in are not specified.
+  data_product_sector: Technology
+
+  dp_flavor:
+    name: Zero-concentrated DP
+    # input_metric: ''
+    # bound_on_input_distance: ''
+    # output_measure: ''
+    # bound_on_output_distance: ''
+    data_domain: Local training cache consisting of user keyboard input from eligible devices of Spanish-language Gboard users.
+    unprotected_quantities: There is no mention of unprotected quantities.
+
+  privacy_loss:
+    privacy_unit: |
+      User-level
```

Comment:
A little in the weeds, but given the description below, this looks like "Device-level" and in many industry settings, device-level is considered slightly different than User-level.  User-level is when devices can be tied together with one user login, and device-level is when the same user of k devices (say, phone and laptop, or phone and multiple browsers) gives k signals.  Probably a discussion for the group about how/when/if we make that distinction.